rma/models/stock_picking.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2020 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    rma_count = fields.Integer(
+        string="RMA count",
+        compute="_compute_rma_count",
+    )
+
+    def _compute_rma_count(self):
+        for rec in self:
+            rec.rma_count = len(rec.move_ids.mapped("rma_ids"))
+
+    def copy(self, default=None):
+        self.ensure_one()
+        if self.env.context.get("set_rma_picking_type"):
+            location_dest_id = default["location_dest_id"]
```

Comment:
We can't rely on this key as it isn't always present: https://github.com/odoo/odoo/blame/5f1ad0c1e4c1e8ed41337ccd4c12e70c9c760c85/addons/stock/wizard/stock_picking_return.py#L124

This provokes an error when you try to create an RMA from the picking return wizard when storage locations aren't active

Reply:
OK, fixed, but this may be a symptom of other bigger thing. Please try again.

Reply:
> this may be a symptom of other bigger thing

Yes, looks like we don't have those values anymore when the group  removes the field. Related:
https://github.com/OCA/rma/pull/361#discussion_r1305510978